django/restapi/fixtures/user.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "model": "auth.user",
+    "pk": 1,
+    "fields": {
+      "username": "k.müller",
+      "first_name": "Kevin",
+      "last_name": "Müller",
+      "password": "1234"
+    }
+  },
+  {
+    "model": "auth.user",
+    "pk": 2,
+    "fields": {
+      "username": "dHaus",
+      "first_name": "David",
+      "last_name": "Haus",
+      "password": "1234",
```

Comment:
```suggestion
      "password": "pbkdf2_sha256$320000$weVQAcFLezx0glHVvHyf5G$v8d6rFBAqdtwpS2ndWqOKCVqFrnbgnH86/LnZJueGTc=",
```

Reply:
Yeah works now, how did you get the hash, so we can include it as a comment?

Reply:
I added them via `/register` and looked in the database